fuseki/shared-dimensions.trig

```diff
@@ -13,8 +13,11 @@ base <https://ld.admin.ch/cube/dimension/>
 
 <shared-dimensions> a void:Dataset .
 
-graph <https://register.ld.admin.ch/opendataswiss/categories> {
-  <https://register.ld.admin.ch/opendataswiss/category/agriculture> a schema:DefinedTerm .
+graph <https://lindas.admin.ch/sfa/opendataswiss> {
+  <https://register.ld.admin.ch/opendataswiss/category/agriculture> a schema:DefinedTerm ;
+    schema:inDefinedTermSet <https://register.ld.admin.ch/opendataswiss/category> ;
+    schema:name "Agriculture"@en ;
+  .
```

Comment:
This is required so that the category appears in the dataset form.

Reply:
And it wasn't until now?

Reply:
No because the query only returns categories with the proper type and with a name.

It was working in the past because we were retrieving these categories from `test.lindas.admin.ch`, but not anymore since you merged the Hierarchies PR ;)